recipes/sfml/3.x/conanfile.py

```diff
@@ -0,0 +1,253 @@
+            del self.options.opengl
+
+        # As per CMakeLists.txt#L44, Android is always shared
+        if self.settings.os == "Android":
+            del self.options.shared
+            del self.options.fPIC
+            self.package_type = "shared-library"
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.window:
+            if self.settings.os in ["Linux", "FreeBSD"]:
+                self.requires("xorg/system")
+                if self.settings.os == "Linux":
+                    self.requires("libudev/system")
+
+            if self.settings.os not in ("iOS", "Android"):  # Handled as a framework
+                self.requires("opengl/system")
+            self.requires("vulkan-headers/[*]")
+
+        if self.options.graphics:
+            if self.settings.os == "Android" or self.settings.os == "iOS":
+                self.requires("zlib/[>=1.2.11 <2]")
+            if self.settings.os == "iOS":
+                self.requires("bzip2/1.0.8")
+            self.requires("freetype/2.13.2")
+            self.requires("stb/[*]")
+
+        if self.options.audio:
+            self.requires("vorbis/1.3.7")
+            self.requires("flac/1.4.3")
+            self.requires("minimp3/cci.20211201")
+            self.requires("miniaudio/0.11.22", transitive_headers=True)
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.24]")
+
+    def validate(self):
+        check_min_cppstd(self, 17)
+
+        if self.options.graphics and not self.options.window:
+            raise ConanInvalidConfiguration(f"-o={self.ref}:graphics=True requires -o={self.ref}:window=True")
+
+        if self.options.get_safe("shared") and is_msvc_static_runtime(self):
+            raise ConanInvalidConfiguration(f"{self.ref} does not support shared libraries with static runtime")
+
+    def validate_build(self):
+        if self.settings.os == "Macos" and self.settings.compiler != "apple-clang":
+            raise ConanInvalidConfiguration(f"{self.ref} is not supported on {self.settings.os} with {self.settings.compiler}")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        apply_conandata_patches(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        tc.cache_variables["SFML_BUILD_WINDOW"] = self.options.window
+        tc.cache_variables["SFML_BUILD_GRAPHICS"] = self.options.graphics
+        tc.cache_variables["SFML_BUILD_NETWORK"] = self.options.network
+        tc.cache_variables["SFML_BUILD_AUDIO"] = self.options.audio
+
+        if self.options.window:
+            tc.cache_variables["SFML_OPENGL_ES"] = self.options.opengl == "es"
+
+        tc.cache_variables["SFML_GENERATE_PDB"] = False  # PDBs not allowed in CCI
+
+        if self.settings.os == "Windows":
+            tc.cache_variables["SFML_USE_STATIC_STD_LIBS"] = is_msvc_static_runtime(self)
```

Comment:
Sorry for the noise, but I saw that this `SFML_USE_STATIC_STD_LIBS` could also apply to Win  + GCC (https://github.com/SFML/SFML/blob/c25fb2b494cd737093eb4a9df2d74d3c4c026af5/cmake/Macros.cmake#L23), so maybe we have to change that condition and make it wider 😅 @AbrilRBS @perseoGI